tests/foreman/cli/test_remoteexecution.py

```diff
@@ -254,39 +245,36 @@ class RemoteExecutionTestCase(CLITestCase):
     """Implements job execution tests in CLI."""
 
     @classmethod
-    @skip_if_not_set('clients', 'fake_manifest')
+    @skip_if_not_set('clients', 'fake_manifest', 'vlan_networking')
     def setUpClass(cls):
         """Create Org, Lifecycle Environment, Content View, Activation key
         """
         super(RemoteExecutionTestCase, cls).setUpClass()
-        cls.org = make_org()
+        cls.org = entities.Organization().create()
         ssh.command(
             '''echo 'getenforce' > {0}'''.format(TEMPLATE_FILE)
         )
-        cls.env = make_lifecycle_environment({
-            u'organization-id': cls.org['id'],
-        })
-        cls.content_view = make_content_view({
-            u'organization-id': cls.org['id'],
-        })
-        cls.activation_key = make_activation_key({
-            u'lifecycle-environment-id': cls.env['id'],
-            u'organization-id': cls.org['id'],
-        })
-        # Add subscription to Satellite Tools repo to activation key
-        setup_org_for_a_rh_repo({
-            u'product': PRDS['rhel'],
-            u'repository-set': REPOSET['rhst7'],
-            u'repository': REPOS['rhst7']['name'],
-            u'organization-id': cls.org['id'],
-            u'content-view-id': cls.content_view['id'],
-            u'lifecycle-environment-id': cls.env['id'],
-            u'activationkey-id': cls.activation_key['id'],
-        })
+        # create subnet for current org, default loc and domain
+        cls.sn = entities.Subnet(
+                domain=[1],
+                gateway=settings.vlan_networking.gateway,
+                ipam='DHCP',
+                location=[2],
+                mask=settings.vlan_networking.netmask,
+                network=settings.vlan_networking.subnet,
+                organization=[cls.org.id]
+            ).create()
+        # add rex proxy to subnet, default is internal proxy (id 1)
+        bug_opened = bz_bug_is_open(1328322)
+        if bug_opened:
```

Comment:
Nitpick: Since there is only non-conditional else, we can move `bz_bug_is_open` to `if` line.